src/hb-ot-maxp-table.hh

```diff
@@ -83,46 +119,27 @@ struct maxp
 
   static inline void drop_hint_fields (hb_subset_plan_t *plan, OT::maxp *maxp_prime)
   {
+    maxpV1Tail &v1 = StructAfter<maxpV1Tail> (*maxp_prime);
```

Comment:
This should go after the version check (inside the block).

Reply:
Done.